src/OpenSage.Game/Network/SkirmishManager.cs

```diff
@@ -0,0 +1,181 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Net;
+using System.Net.Sockets;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Linq;
+using ProtoBuf;
+
+namespace OpenSage.Network
+{
+    public class SkirmishManager
+    {
+        public class SkirmishClient
+        {
+            TcpClient Client;
```

Comment:
Why are there two clients? This is unused

Reply:
At the moment, it hasn't even been run once, on my machine that is :D So, it's well possible that there is extraneous code :)